docs.go

```diff
@@ -146,3 +147,35 @@ func flagDetails(flag DocGenerationFlag) string {
 	}
 	return ": " + description
 }
+
+func prepareUsageText(command *Command) string {
+	usageText := ""
+	if command.UsageText != "" {
+		// Remove leading and trailing newlines
+		preparedUsageText := strings.TrimSuffix(command.UsageText, "\n")
+		preparedUsageText = strings.TrimPrefix(preparedUsageText, "\n")
+
+		if strings.Contains(preparedUsageText, "\n") {
+			// Format multi-line string as a code block
+			usageText = fmt.Sprintf("```\n%s\n```\n", preparedUsageText)
+		} else {
+			// Style a single line as a note
+			usageText = fmt.Sprintf(">%s\n", preparedUsageText)
+		}
+	}
+	return usageText
+}
+
+func prepareUsage(command *Command, usageText string) string {
+	usage := ""
+	if command.Usage != "" {
```

Comment:
Since both code blocks depend on `command.Usage` not being empty, we have the same situation as `prepareUsageText`, where the entire business logic of the function is nested under a single conditional branch. 

We can simplify the entire body to this:

```go
if command.Usage == "" {
  return ""
}

usage := command.Usage + "\n"
if usageText != "" {
  usage += "\n"
}

return usage
```